app/helpers/template_helpers.py

```diff
@@ -144,9 +144,21 @@ def footer_context(self) -> dict[str, Any]:
             context["footerWarning"] = self._footer_warning
 
         if self._survey_config.footer_links:
-            context["rows"] = [{"itemsList": self._survey_config.footer_links}]
-
-        if self._survey_config.footer_legal_links:
+            if self._survey_type:
+                context["rows"] = [{"itemsList": self._survey_config.footer_links}]
+
+            else:
+                context["rows"] = [
+                    {
+                        "itemsList": [
+                            footer_link
+                            for footer_link in self._survey_config.footer_links
+                            if "Contact us" not in footer_link.values()
```

Comment:
Could potentially do this the other way around so it is more explicit, i.e check for the values we want rather than the not in?

Reply:
So I inverted the logic and now we see the fields that we want to keep:
```python
for footer_link in self._survey_config.footer_links:
      items_list.extend(
          footer_link
          for item in [
              "What we do",
              "Accessibility",
              "Help",
              "Languages",
              "BSL and audio videos",
          ]
          if item in footer_link.values()
      )

      context["rows"] = [{"itemsList": items_list}]
```
Thanks!